cmd/list.go

```diff
@@ -767,18 +758,16 @@ var listOrganizationDeployTargetsCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		organizationName, err := cmd.Flags().GetString("name")
+		organizationName, err := cmd.Flags().GetString("organization")
 		if err != nil {
 			return err
 		}
 		organizationID, err := cmd.Flags().GetUint("id")
 		if err != nil {
 			return err
 		}
-		if err := requiredInputCheck("Organization name", organizationName); err != nil {
-			if err := requiredInputCheck("Organization ID", strconv.Itoa(int(organizationID))); err != nil {
-				return err
-			}
+		if organizationName == "" && organizationID == 0 {
+			return fmt.Errorf("Missing arguments: Organization name or ID is not defined")
```

Comment:
All `fmt.Errorf` should have the uppercase word starting the error changed to lower case. This will reduce some warnings that go tools will present in code editors and follows go formatting standards which we have been a bit loose with in the past